cpp/include/cudf/utilities/traits.hpp

```diff
@@ -505,6 +504,8 @@ constexpr inline bool is_fixed_width(data_type type)
   return cudf::type_dispatcher(type, is_fixed_width_impl{});
 }
 
+class string_view;
+
```

Comment:
cudf/types.hpp already has forward declaration of cudf::string_view. This forward declaration may not be needed.

Reply:
Maybe, but I feel this makes the header file more complete/clear on its own.